python/pyspark/pandas/tests/data_type_ops/test_string_ops.py

```diff
@@ -217,6 +220,20 @@ def test_from_to_pandas(self):
     def test_isnull(self):
         self.assert_eq(self.pser.isnull(), self.psser.isnull())
 
+    def test_astype(self):
+        pser = self.pser
+        psser = self.psser
+
+        # [x, y, z, <NA>] is returned in pandas
```

Comment:
Shall we add `TODO(SPARK-35976):` here as well?

Reply:
For sure, added.